index.html

```diff
@@ -303,6 +303,32 @@ <h2 class="refer-link">LET'S CONNECT</h2>
                 </div>
             </div>
         </section>
-       </main>
+        <section id="contact-container">
+            <div id="contact-items">
+                <div id="description-contact-box">
+                    <h1 class="contact-title-1">Contact me</h1>
+                    <p class="contact-description-1">
+                        If you have an application you are interested in developing, a
+                        feature that you need built or a project that needs coding. I’d
+                        love to help with it.
+                    </p>
+                </div>
+                <form id="input-contact-box" action="https://formspree.io/f/xpzbqlab" method="post">
+                    <label class="label" for="username"></label>
+                    <input id="username" class="input-i" name="username" type="text" maxlength="30"
+                        placeholder="Emmnanuel Paul" required="required" />
+                    <label class="label" for="email"></label>
+                    <input id="email" class="input-i" name="email" type="email" value="" placeholder="emmapaul247@gmail.com"
+                        required="required" />
+                    <label class="label" id="input-message" for="msg"></label>
+                    <textarea class="input-m" name="msg" id="msg" maxlength="500" cols="20" rows="10"
+                        placeholder="Write your message here" required="required"></textarea>
+                    <button class=" get-touch-btn" type="submit">Get In Touch</button>
+                   </form>
+                   <hr/>
```

Comment:
- [ ] Kindly delete this horizontal line as it is not required in your design 👍🏿